proprietary/tokens/src/components/ams/alert.tokens.json

```diff
@@ -9,7 +9,7 @@
       "severity-indicator": {
```

Comment:
Can you move this to `packages-proprietary`?

Reply:
Yes, merging develop resolved that.